src/ontology/bridge/cl-bridge-to-ehdaa2.owl

```diff
@@ -7,20 +7,13 @@
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
      xmlns:xml="http://www.w3.org/XML/1998/namespace"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
-     xmlns:doap="http://usefulinc.com/ns/doap#"
-     xmlns:foaf="http://xmlns.com/foaf/0.1/"
-     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
-     xmlns:terms="http://purl.org/dc/terms/"
-     xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#">
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
     <owl:Ontology rdf:about="http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-ehdaa2.owl">
-        <owl:versionIRI rdf:resource="http://purl.obolibrary.org/obo/uberon/bridge/cl-bridge-to-ehdaa2/2023-09-05/uberon/bridge/cl-bridge-to-ehdaa2.owl"/>
-        <dc:contributor rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Jonathan Bard</dc:contributor>
-        <dc:creator rdf:datatype="http://www.w3.org/2001/XMLSchema#string">CL editors</dc:creator>
-        <dc:description rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Equivalence axioms between EHDAA2 and CL. EHDAA2 replaces EHDAA</dc:description>
-        <dc:title rdf:datatype="http://www.w3.org/2001/XMLSchema#string">CL bridge to ehdaa2</dc:title>
-        <terms:references rdf:resource="http://purl.obolibrary.org/obo/cl.owl"/>
-        <terms:references rdf:resource="http://purl.obolibrary.org/obo/ehdaa2.owl"/>
-        <rdfs:seeAlso rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">http://purl.obolibrary.org/obo/uberon/references/reference_0000026</rdfs:seeAlso>
+        <owl:versionIRI rdf:resource="http://purl.obolibrary.org/obo/uberon/releases/2023-09-07/bridge/cl-bridge-to-ehdaa2.owl"/>
+        <dc:contributor>Jonathan Bard</dc:contributor>
```

Comment:
> I only observe here that this is using the wrong dc namespace.

This is using the `http://purl.org/dc/elements/1.1/` namespace, *as in the original bridges*. No objection to switching to `http://purl.org/dc/terms/`, but that will require a change in the SSSOM plugin.

Reply:
The change [has been made](https://github.com/gouttegd/sssom-java/commit/265855629f9a87fad71b816587b81d6a6285000b) in sssom-java, so this will be automatically fixed when 1) a new sssom-java release will be available and 2) the Uberon Makefile will be updated to use that new release.